python/lsst/ts/athexapod/csc.py

```diff
@@ -154,18 +158,18 @@ async def configure(self, config):
         config
         """
         host = config.host
+        # Support teststand connection via Environment Variable.
         if host.startswith("ENV."):
             env_name = host.split("ENV.")[1]
             if env_name not in os.environ:
-                raise RuntimeError(
-                    "Failed to configure CSC."
-                    f"Environment variable {env_name} is not defined."
-                )
+                msg = f"""Failed to configure CSC
```

Comment:
This seems like a pretty innocuous change. Please, revert back to the original code.